ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh

```diff
@@ -72,6 +71,16 @@ export BOOTSTRAP_URI
 # begin bootstrapping
 if openshift-install coreos print-stream-json 2>/tmp/err.txt >coreos.json; then
   RHCOS_AMI="$(jq -r --arg region "$AWS_REGION" '.architectures.x86_64.images.aws.regions[$region].image' coreos.json)"
+  BOOTSTRAP_INSTANCE_TYPE=i3.large
+  MASTER_INSTANCE_TYPE=m5.xlarge
+  WORKER_INSTANCE_TYPE=m5.xlarge
+  if [[ "${CLUSTER_TYPE}" == "aws-arm64" ]]; then
+    RHCOS_AMI="$(jq -r --arg region "$AWS_REGION" '.architectures.aarch64.images.aws.regions[$region].image' coreos.json)"
+    BOOTSTRAP_INSTANCE_TYPE=m6g.large
+    MASTER_INSTANCE_TYPE=m6g.xlarge
+    WORKER_INSTANCE_TYPE=m6g.xlarge
```

Comment:
I'm not 100% certain about this, but can't these variables be set in the job definition or elsewhere in the config? And then we could set the default values in the step config? That would be better than handling this in code.